autoload/committia/git.vim

```diff
@@ -170,11 +181,16 @@ function! committia#git#diff() abort
         return ['']
     endif
 
+    " Fugly hack to tell committia#git#status() to get the status from the
+    " commit message template too, otherwise status may not match with diff
+    " Could be removed if g:committia#git#use_verbose was enabled by default
+    let s:use_verbose_status = 1
+
     return getline(line, '$')
 endfunction
 
 function! s:diff_start_line() abort
-    let re_start_diff_line = '# -\+ >8 -\+\n\%(#.*\n\)\+diff --git'
+    let re_start_diff_line = '^[#;@!$%^&|:] -\+ >8 -\+\n\%([#;@!$%^&|:].*\n\)\+diff --git'
```

Comment:
Quetion: When the characters other than `#` are used at the head of line?

Reply:
Yes, which happens when `core.commentChar` is set to something other than `#` in git config. You can test this using `git -c core.commentChar=% commit --amend --verbose`. Before this change committia will not find the scissors line and create splits for diff and status windows, after the change it works as expected.

Reply:
Thanks. I didn't notice the `core.commentChar` config.

Reply:
Actually, this doesn't work quite as expected even with this change, as `committia#git#end_of_edit_region_line()` also assumes `core.commentChar` is `#`. I'll have a look at fixing that too.

Reply:
Fixed `committia#git#end_of_edit_region_line()` in 024a5c77